src/types/entities.ts

```diff
@@ -30,3 +30,78 @@ export interface User {
   birthday: string | null;
   bioHtml: string | null;
 }
+
+export enum WorkRatings {
```

Comment:
Missing 'Not Rated' rating

Reply:
Done. Should we remove `null` from `rating: WorkRatings | null;` below?